addons/test_mail/tests/test_message_track.py

```diff
@@ -39,6 +43,77 @@ def test_message_track_author(self):
 
         self.assertEqual(self.record.message_ids.author_id, self.partner_admin)
 
+    def test_message_track_default_message(self):
+        record = self.track_record
+        with self.mock_mail_gateway():
+            record.user_id = self.user_admin
+            self.flush_tracking()
+
+        messages = record.message_ids
+        self.assertEqual(len(messages), 1)
+        self.assertEqual(messages.body, '<p>There was a change on Test for fields "user_id"</p>',
+                         'Default message should be used')
+
+        with self.mock_mail_gateway():
+            record._track_set_log_message('Hi')
+            record.user_id = False
+            self.flush_tracking()
+
+        messages = record.message_ids - messages
+        self.assertEqual(len(messages), 1)
+        self.assertEqual(messages.body, '<p>Hi</p>', '_track_set_log_message should take priority over default message')
+
+    def test_track_formatting(self):
+        field_dname = 'Responsible'
+        field_name = 'user_id'
+        field_type = 'many2one'
+        original_user = self.user_admin
+        new_user = self.user_employee
+
+        records = self.env['mail.test.track'].create([
+            {'name': f'Test{n}',
+             'user_id': original_user.id,
+             'track_hidden_fields': 'user_id' if n == 0 else ''
+             } for n in range(2)])
```

Comment:
Isn't it possible to create two records manually ^^ ? Seems it would be easier to understand what you want to do :)